### PR TITLE
PR with proposed Second Screen Community Group Charter and requirements document

### PR DESCRIPTION
**Pull request by markafoltz**

### CODE REVIEW

**Comment by anssiko** [comment]
Thanks @mfoltzgoogle. We'll discuss this proposal on Wednesday (exact slot TBD), see: https://www.w3.org/wiki/Second_Screen/Meetings/May_2016_F2F#Agenda

**Comment by anssiko** [comment]
We've (Intel) reviewed the proposal and it looks good with some suggestions for improvements:
- Clarification to the Out of Scope section: https://github.com/mfoltzgoogle/cg-charter/pull/3
- Match the latest W3C CG Charter template: https://github.com/mfoltzgoogle/cg-charter/pull/2

**Comment by markafoltz** [comment]
Update:

The current state of the pull request reflects the PR from @anssiko and feedback from the working group in the F2F (both resolutions, and suggestions mentioned in the minutes.)
- This reflects a narrowed scope of deliverables, which now include only specifications and exclude prototype implementations.
- It no longer lists specific normative reports.
- It also does not mandate a specific factoring of the specifications.
- It is specific about supporting the "media flinging" (via `src=` or `<source>`) use case for Remote Playback API.

I did add a scope item to support protocol extensions to allow for adding features that would currently be out of scope via future specifications.

Connection via protocols other than IPv4/IPv6 is explicitly not in scope; this may affect applicability for scenarios like FlyWeb in the future.  Want to get @sicking's input on that.

I am working on incorporating the updated charter template into the draft, but not ready yet.   If there are no objections to the content of this updated PR, I will request to merge it, and handle the template language in a separate request.

**Comment by sicking** [comment]
I definitely don't think we should expand any scope in order to accommodate FlyWeb. I have no idea where or when FlyWeb will get standardized, and so assuming that it'll happen in the second-screen WG seems premature.

That said, I think BLE would be a great fit for a backend for the presentation API. If the details of that is worked on in this WG, or elsewhere, is not something that I have an opinion on. And likely it would not be appropriate to work on that until there are interest from implementations to implement.

File: index.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
-<!DOCTYPE html>
```
Comment (anssiko) [inline index.html:1]: I'd prefer HTML DOCTYPE over XHTMLs.

File: index.html [hunk 2]
```diff
@@ -40,13 +52,13 @@
         Start Date:
       </dt>
       <dd>
-        DD MMM YYYY
+        17 Nov 2015
```
Comment (anssiko) [inline index.html]: I guess 2016.

File: index.html [hunk 3]
```diff
@@ -196,65 +313,172 @@ <h2 id="liaisons">
...
+      This is explicitly allowed (and preferred to blocking progress) 
+      with a goal of letting implementation experience inform which 
+      spec is ultimately chosen by the group to move ahead with.
     </p>
+    
+    <p>
+      Any decisions reached at any meeting are tentative and should be 
+      recorded in a GitHub Issue for groups that use GitHub and otherwise 
+      on the group's public mail list. Any group participant may object to 
+      a decision reached at an online or in-person meeting within 7 days of
```
Comment (anssiko) [inline index.html]: I'd be fine with 10 to match the WG's window.

**Comment by anssiko** [comment]
@mfoltzgoogle Thanks! I added some comments. Would you address them and then I'd merge this PR.

**Comment by markafoltz** [comment]
@anssiko Thanks, I've incorporated your suggestions into the current PR.  In addition, I removed the notes left by @wcarr. The following changes were also made to mostly follow the template with a few exceptions.
- Adds "Work Limited" paragraph from the template.
- Edits so that GitHub is the repository of record for Contributions.
- Removes mentions of RFC 2777 for voting for Chair election, instead consulting with the Community Development Lead for the appropriate process or tool.

**Comment by wayneca** [comment]
I'll get that RFC 2777 change into the cg template

**Pull request merged**